jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -1,11 +1,15 @@
 def run(params) {
     timestamps {
+        //Capybara configuration
+        def capybara_timeout = 60
```

Comment:
No please. You are overwritting and not taking into account the jenkins values passed by parameter when running the BV.
We should not have it here as constants. Starting by the fact that not all environments performs the same way.

Reply:
It was asked by the team to remove some of the parameters we have in the configuration. capybara timeout is not really something we change. If we want a specific timeout depending on platform, it should be added into manager-4.3-qe-build-validation-PRV